### DISCUSSION TITLE
Not able to access https://github.com/SalesforceCommerceCloud/storefront-reference-architecture

### DISCUSSION THREAD

**Discussion by brijesh-born2022** [community/community#33693] [Other Feature Feedback, Questions, & Ideas]
Hi Team,

Due to lost of my mobile and Two factor Authentication, I was not able to login into github account.
So, I have requested to Github support team for removing two factor authentication.
 
 So they disable the two factor authentication and also Removed from the "Salesforce CommerceCloud" organization.
 Now, I'm not able to access https://github.com/SalesforceCommerceCloud/storefront-reference-architecture again.
 I have enabled Two factor authentication again but still not able to access.
Please see attached screen-shot.
![salesforce_disable](https://user-images.githubusercontent.com/107907221/191504009-b94984af-cb8a-468c-b9e0-4b582b139bb3.png)

**Comment by airtower-luna** [comment]
If you've been removed from the organization you'll have to be added to it again. Contact an organization admin for that. I don't know that organization, but since you were part of it I hope you know who to ask. :sweat_smile:
